extensions/google-chrome/src/google-chrome-new-tab.tsx

```diff
@@ -0,0 +1,229 @@
+import { ActionPanel, popToRoot, closeMainWindow, CopyToClipboardAction, getPreferenceValues, Icon, List, OpenInBrowserAction, showToast, ToastStyle} from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+import { HistoryEntry, useChromeHistorySearch } from "./browserHistory"
+import { useEffect, useState, ReactElement} from "react";
+import { faviconUrl } from "./utils"
+
+class Tab {
+  static readonly TAB_CONTENTS_SEPARATOR: string = '~~~';
+
+  constructor(
+    public readonly title: string,
+    public readonly url: string,
+    public readonly favicon: string,
+    public readonly windowsIndex: number,
+    public readonly tabIndex: number
+  ) { }
+
+  static parse(line: string): Tab {
+    const parts = line.split(this.TAB_CONTENTS_SEPARATOR);
+
+    return new Tab(parts[0], parts[1], parts[2], +parts[3], +parts[4]);
+  }
+
+  key(): string {
+    return `${this.windowsIndex}${Tab.TAB_CONTENTS_SEPARATOR}${this.tabIndex}`;
+  }
+
+  urlWithoutScheme(): string {
+    return this.url.replace(/(^\w+:|^)\/\//, '').replace('www.', '');
+  }
+
+
+  googleFavicon(): string {
+    return faviconUrl(64, this.url)
+  }
+}
+
+async function getOpenTabs(useOriginalFavicon: boolean): Promise<Tab[]> {
+  const faviconFormula = useOriginalFavicon
+    ? `execute of tab _tab_index of window _window_index javascript ¬
+                    "document.head.querySelector('link[rel~=icon]').href;"`
+    : '""'
+
+  const openTabs = await runAppleScript(`
+      set _output to ""
+      tell application "Google Chrome"
+        set _window_index to 1
+        repeat with w in windows
+          set _tab_index to 1
+          repeat with t in tabs of w
+            set _title to get title of t
+            set _url to get URL of t
+            set _favicon to ${faviconFormula}
+            set _output to (_output & _title & "${Tab.TAB_CONTENTS_SEPARATOR}" & _url & "${Tab.TAB_CONTENTS_SEPARATOR}" & _favicon & "${Tab.TAB_CONTENTS_SEPARATOR}" & _window_index & "${Tab.TAB_CONTENTS_SEPARATOR}" & _tab_index & "\\n")
+            set _tab_index to _tab_index + 1
+          end repeat
+          set _window_index to _window_index + 1
+          if _window_index > count windows then exit repeat
+        end repeat
+      end tell
+      return _output
+  `);
+
+  return openTabs.split("\n").filter(line => line.length !== 0).map(line => Tab.parse(line));
+}
+
+async function openNewTab(queryText: string | null | undefined): Promise<boolean | string> {
+  popToRoot();
+  closeMainWindow({ clearRootSearch: true });
+  
+  const script = `
+    tell application "Google Chrome"
+      activate
+      tell window 1
+          set newTab to make new tab ` + (queryText ? 'with properties {URL:"http://www.google.com/search?q='+queryText+'"}' : '') + ` 
```

Comment:
```suggestion
          set newTab to make new tab ` + (queryText ? 'with properties {URL:"https://www.google.com/search?q='+queryText+'"}' : '') + ` 
```
HTTPS should be used here 🏎️